daemon/exec.go

```diff
@@ -287,6 +287,20 @@ func (d *Daemon) ContainerExecStart(ctx context.Context, name string, stdin io.R
 	return nil
 }
 
+// ContainerExecKill kills an exec instance.
+func (d *Daemon) ContainerExecKill(ctx context.Context, id string, config *types.ExecKillConfig) error {
+	e, err := d.getExecConfig(id)
+	if err != nil {
+		return err
+	}
+
+	sig := int(signal.SignalMap["TERM"])
+	if config.Force {
+		sig = int(signal.SignalMap["KILL"])
+	}
+	return d.containerd.SignalProcess(ctx, e.ContainerID, id, sig)
```

Comment:
what happens on Windows?

Reply:
I think this method supports signals on Windows.
https://github.com/moby/moby/blob/98dcded647d9f13860266b24683ca65eef5060f6/pkg/signal/signal_windows.go#L23-L26

Reply:
Here's the signal map of containerd.

https://github.com/containerd/containerd/blob/b02ab6c742a48d5ea0ebadaa3c18329356a383a4/signal_map_windows.go#L25-L39